addon/mixins/attachable.js

```diff
@@ -32,12 +32,12 @@ export default Ember.Mixin.create({
       if (!Ember.isNone(data[key])) {
         if (Ember.isArray(data[key])) {
           return data[key].forEach(function(val) {
-            return formData.append("" + root + "[" + key + "][]", val);
+            return formData.append("" + root + "[" + key + "][]", val, val.filename || 'blob');
```

Comment:
@timiyay This will add `val.filename || 'blob'` to all the data inside the formData. We should only do this to the attachment files.